tests/foreman/ui/test_oscapcontent.py

```diff
@@ -0,0 +1,95 @@
+from robottelo.constants import (
+    ANY_CONTEXT,
+    OSCAP_CONTENT_FILE,
+    OSCAP_DEFAULT_CONTENT,
+)
+from robottelo.datafactory import valid_data_list
+from robottelo.helpers import get_data_file
+from robottelo.test import UITestCase
+from robottelo.ui.factory import set_context, make_oscapcontent
+from robottelo.ui.session import Session
+
+
+class OpenScapContent(UITestCase):
+    """Implements Oscap Content tests in UI."""
+
+    @classmethod
+    def setUpClass(cls):
+        super(OpenScapContent, cls).setUpClass()
+        cls.content_path = get_data_file(OSCAP_CONTENT_FILE)
+
+    def test_positive_create_oscapcontent(self):
+        """@Test: Create OpenScap content.
+
+        @Feature: OpenScap - Positive Create.
+
+        @Steps:
+
+        1. Create an openscap content.
+        2. Provide all the appropriate parameters.
+
+        @Assert: Whether creating  content for OpenScap is successful.
+
+        """
+        with Session(self.browser) as session:
+            for content_name in valid_data_list():
+                with self.subTest(content_name):
+                    set_context(session, org=ANY_CONTEXT['org'])
+                    session.nav.go_to_oscap_content()
+                    make_oscapcontent(
+                        session,
+                        name=content_name,
+                        content_path=self.content_path,
+                    )
+                    self.assertIsNotNone(
+                        self.oscapcontent.search(content_name))
+
+    def test_positive_default_oscapcontent(self):
+        """@Test: Check whether OpenScap content exists by default.
+
+        @Feature: OpenScap - Test Default Content.
+
+        @Steps:
+
+        1. Navigate to oscap Content page.
```

Comment:
I have run `make docs-clean docs` and checked for oscapcontent related WARNINGS. I fixed whichever I could, but there are still 4 more warnings.

@anyone: Please feel free to fix it, in another PR if you can quickly. :)


```
(robottelo)[xyz@abc robottelo]$ make docs-clean docs
make[1]: Entering directory '~/git_world/robottelo/docs'
rm -rf _build/*
make[1]: Leaving directory '~/git_world/robottelo/docs'
make[1]: Entering directory '~/git_world/robottelo/docs'
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.3.1
making output directory...
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 19 source files that are out of date
updating environment: 19 added, 0 changed, 0 removed
reading sources... [100%] reviewing_PRs                                                                                                                                                                            
~/git_world/robottelo/robottelo/helpers.py:docstring of robottelo.helpers.install_katello_ca:4: WARNING: Field list ends without a blank line; unexpected unindent.
~/git_world/robottelo/robottelo/helpers.py:docstring of robottelo.helpers.remove_katello_ca:4: WARNING: Field list ends without a blank line; unexpected unindent.
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] reviewing_PRs                                                                                                                                                                             
~/git_world/robottelo/robottelo/datafactory.py:docstring of robottelo.datafactory.invalid_values_list:None: WARNING: more than one target found for cross-reference u'list': robottelo.cli.docker.DockerRegistry.list, robottelo.cli.docker.DockerImage.list, robottelo.cli.docker.DockerTag.list, robottelo.cli.lifecycleenvironment.LifecycleEnvironment.list, robottelo.cli.docker.DockerContainer.list, robottelo.cli.base.Base.list
~/git_world/robottelo/docs/api/tests.foreman.cli.rst:37: WARNING: py:class reference target not found: Domain
generating indices... genindex py-modindex
highlighting module code... [100%] robottelo.datafactory                                                                                                                                                           
writing additional pages... search
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 4 warnings.

Build finished. The HTML pages are in _build/html.
make[1]: Leaving directory '~/git_world/robottelo/docs'
```